src/main/webapp/app/upload/upload.component.ts

```diff
@@ -12,12 +19,84 @@ export class UploadComponent {
     error: string;
     uploadResponse = { status: '', message: '' };
 
+    currentPool: Map<String, Pool> = new Map<String, Pool>();
+    newPool: Map<String, Pool> = new Map<String, Pool>();
+
+    public poolType = PoolType;
+    public isCollapsed = true;
     public files: UploadFile[] = [];
-    file: File;
 
-    constructor(private uploadService: UploadService, private router: Router) {}
+    public currentConflictPool: Pool[] = [];
+    public newConflictPool: Pool[] = [];
+    public merged: Pool[] = [];
+    public choosePattern = new Map();
+    public file: File;
+
+    constructor(private uploadService: UploadService, private router: Router, private poolService: PoolListService) {}
+
+    async onSubmit() {
+        if (!this.file) {
+            return (this.error = 'Please insert file');
+        }
+        this.clearField();
+        const formData = new FormData();
+        formData.append('file', this.file);
+
+        const oldOnesPromise = this.poolService.getPools().toPromise();
```

Comment:
I think this variables shouldn't be consts, this is the issue when user wants to change file to upload after conflict table is shown. It reloads table, but content stays the same.